google/cloud/security/inventory/pipelines/load_gke_pipeline.py

```diff
@@ -0,0 +1,129 @@
+        gke_services = {}
+        for project in projects:
+            clusters = self.safe_api_call('get_clusters', project.id)
+
+            if clusters:
+            
+                for cluster in clusters:
+                    self_link_parts = cluster.get('selfLink').split('/')
+                    zone = self_link_parts[self_link_parts.index('zones')+1]
+                    server_config = self.safe_api_call(
+                        'get_serverconfig', project.id, zone)
+                    cluster['serverConfig'] = server_config
+
+                gke_services[project.id] = clusters
+
+
+        return gke_services
+    # pylint: enable=too-many-locals, too-many-nested-blocks
+
+    def _transform(self, resource_from_api):
+        """Create an iterator of GKE services to load into database.
+
+        Args:
+            resource_from_api (dict): GKE services from GCP API, keyed by
+                project id, with list of clusters as values.
+            
+            {project1: [clusters],
+             project2: [clusters],
+             project3: [clusters]}
+
+            Each cluster has additional server_config data included.
+
+        Yields:
+            iterator: GKE service in a dict.
+        """
+        for project_id, clusters in resource_from_api.iteritems():
+            for cluster in clusters:
+                yield {'project_id': project_id,
+                       'addons_config': parser.json_stringify(
+                            cluster.get('addonsConfig')),
+                       'cluster_ipv4_cidr': cluster.get('clusterIpv4Cidr'),
+                       'create_time': cluster.get('createTime'),
+                       'current_master_version': cluster.get('currentMasterVersion'),
+                       'current_node_count': cluster.get('currentNodeCount'),
+                       'current_node_version': cluster.get('currentNodeVersion'),
+                       'endpoint': cluster.get('endpoint'),
+                       'initial_cluster_version': cluster.get('initialClusterVersion'),
+                       'instance_group_urls': cluster.get('instanceGroupUrls'),
+                       'legacy_abac': parser.json_stringify(
+                            cluster.get('legacyAbac')),
+                       'locations': cluster.get('locations'),
+                       'logging_service': cluster.get('loggingService'),
+                       'master_auth': parser.json_stringify(
+                            cluster.get('masterAuth')),
+                       'monitoring_service': cluster.get('monitoringService'),
+                       'name': cluster.get('name'),
+                       'network': cluster.get('network'),
+                       'node_config': parser.json_stringify(
+                            cluster.get('nodeConfig')),
+                       'node_ipv4_cidr_size': cluster.get('nodeIpv4CidrSize'),
+                       'node_pools': cluster.get('nodePools'),
```

Comment:
probably need to json.stringify this too.

Reply:
Done.